tests/listengine.js

```diff
@@ -3010,6 +3011,133 @@ describe( 'ListEngine', () => {
 		} );
 	} );
 
+	describe( 'paste integration', () => {
+		it( 'should fix indents of pasted list items', () => {
+			setModelData( modelDoc,
+				'<listItem type="bulleted" indent="0">A</listItem>' +
+				'<listItem type="bulleted" indent="1">B[]</listItem>' +
+				'<listItem type="bulleted" indent="2">C</listItem>'
+			);
+
+			const clipboard = editor.plugins.get( 'clipboard/clipboard' );
+
+			clipboard.fire( 'inputTransformation', {
```

Comment:
I'm hesitating whether it wouldn't be better to test it directly with `insertContent()`. Involving clipboard makes it more integrational but... it's not the best API to simulate pasting. Maybe clipboard should expose a command? I was actually thinking about it initially, but back then it didn't make sense. It'd make sense now.

WDYT?